### PR TITLE
Vet clinic DB : create animals table

### PR DESCRIPTION
**Pull request by MenendezON**
This pull request is about PostgreSQL :

- Create a database;
- Create a table;
- Insert some data;
- Create some queries.

screenshot-2023-09-18

### CODE REVIEW

File: queries.sql [hunk 1]
```diff
@@ -1,3 +1,25 @@
...
+SELECT * FROM animals WHERE date_of_birth BETWEEN '2016-01-01' AND '2019-12-31';
+
+/*List the name of all animals that are neutered and have less than 3 escape attempts.*/
+SELECT * FROM animals WHERE neutered = true AND escape_attempts < 3;
+
+/*List the date of birth of all animals named either "Agumon" or "Pikachu".*/
+SELECT date_of_birth FROM animals WHERE name = 'Agumon' OR name = 'Pikachu';
+
+/*List name and escape attempts of animals that weigh more than 10.5kg*/
+SELECT name, escape_attempts FROM animals WHERE weight_kg >= 10.5;
```
Comment (V-Blaze) [inline queries.sql]: - You have done great so far :+1: But there are a few changes you need to make before moving to the next milestone. :+1: :+1: 
- [ ] This query says `List name and escape attempts of animals that weigh more than 10.5kg`, your current implementation includes even animals that weigh 10.5kg which is not correct, I'll suggest you remove the `=` here. :+1: :wave:

File: queries.sql [hunk 2]
```diff
@@ -1,3 +1,25 @@
-/*Queries that provide answers to the questions from all projects.*/
+/*Queries that provide answers to the questions FROM all projects.*/
```
Comment (V-Blaze) [inline queries.sql:1]: > Take a screenshot of the results of your queries.
Remember to add the screenshot with the results of your queries to your Pull Request description.

- [ ] According to the project requirement, you were supposed to take screenshots of these queries and include them in the pull request description, currently, I can you combine them in a .png format, I'll suggest you also include the screenshot in the pull request description as required and for easy viewing. :+1: :smile:

**Review by V-Blaze — CHANGES_REQUESTED** [review]
## Hi @MenendezON 

## Required changes 🖊️ 
# Almost there 🟡 🟦 🟨 

giphy


Great work so far, You have come a long way, keep moving forward👏 👍  
You've done well implementing the project requirements, but there are still some issues I think you need to improve on. 😄 👋 


 ## Highlights
- Everything that creates the table structure is in the schema.sql file.✔
-  Most queries written correctly👋 
-  Data are correctly inserted into the table✔

 ## Required Changes ♻️

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by TuyishimireEric — APPROVED** [review]
# Status: APPROVED 🟢  
### Hi  @MenendezON   👋 👋 

#### I'm @TuyishimireEric, Your code reviewer,
Good job on implementing the project requirements! 🥇 
done

Your project is complete!  There is nothing else to say other than... it's time to merge it 🦾  
Congratulations! 🎉


## Highlight 👍 
✅ Made suggested changes 


### Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Remember to tag me in your question so I can receive the notification.

------
As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/enus/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think the code review was unfair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).

**Pull request merged**